pytorch_blade/src/compiler/jit/shape_type_spec.cpp

```diff
@@ -162,19 +162,23 @@ ShapeTypeSpec ShapeTypeSpec::GetShapeTypeSpec(
 }
 
 ShapeTypeSpec ShapeTypeSpec::GetShapeTypeSpec(
-    const std::vector<const torch::jit::Value*>& values) {
+    const std::vector<const torch::jit::Value*>& values,
+    bool force_concurrent) {
   std::vector<ShapeType> shape_types;
   shape_types.reserve(values.size());
   for (const auto& val : values) {
-    CHECK(is_concrete_shape_tensor_type(*val));
+    if (force_concurrent)
```

Comment:
Nit: Recommend usages of braces:
```
if (cond) {
  // some codes;
}
```

Reply:
add some description at the function def.